dotcom-rendering/src/web/components/App.tsx

```diff
@@ -1121,7 +1099,6 @@ export const App = ({ CAPI, NAV, ophanRecord }: Props) => {
 			)}
 			<Portal rootId="slot-body-end">
 				<SlotBodyEnd
-					isSignedIn={isSignedIn}
```

Comment:
`isSignedIn` is still available above - isn't it preferable to still pass it in?
Also, `StickyBottomBanner` also takes `isSignedIn` as a prop

Reply:
Ah, good spot, I have also pushed the cookie read down into that component as well.

We do still have `isSignedIn` defined in `App.tsx` so technically we could pass it in but the goal is to remove it from App completely. I'm held back from doing that though until we refactor some other areas first. This is step 1.

Reply:
Just for reference (and I also updated the PR description to include this link) [here is the document](https://docs.google.com/document/d/1kSrZZrJ8hSw6bek5hGa9onE8QDz5TRHFqHV52EOFjts/edit#heading=h.gsod33a0pc47) talking about the goal of removing App.tsx which is what drives this PR